pytorch/inference/docker/1.12/py3/Dockerfile.cpu

```diff
@@ -223,6 +223,8 @@ ARG PYTHON
 
 ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
 
+RUN pip install -U pyOpenSSL cryptography
```

Comment:
are these needed for the tests? what is the reason to update these dockerfiles? If it's not related to the tests, can you just update the description of the PR?

Reply:
Required to fix the issue with version compatibility between pyOpenSSL and cryptography. Docker failing to build past the point of where scikit and pandas are installed via conda.